src/main/java/com/ericsson/ec/subscriptionobject/SubscriptionObject.java

```diff
@@ -0,0 +1,87 @@
+package com.ericsson.ec.subscriptionobject;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import lombok.Getter;
+
+abstract class SubscriptionObject {
+    private static final String SUBSCRIPTION_TEMPLATE_PATH = "src/main/resources/subscriptionsTemplate.json";
```

Comment:
reading the data from this path will not work until we fix the resource reading problem.

Reply:
What should we do in the meanwhile?